flake.nix

```diff
@@ -0,0 +1,45 @@
+{
+    description = "Tool to simplify secure shell connections over AWS simple systems manager.";
+
+    inputs = {
+      nixpkgs.url = "github:NixOS/nixpkgs?ref=nixpkgs-unstable";
+      utils.url = "github:numtide/flake-utils";
+    };
+    outputs = { self, nixpkgs, utils }: {
+      devShell = self.defaultPackage;
+      defaultPackage.x86_64-darwin =
+        with import nixpkgs { system = "x86_64-darwin"; };
```

Comment:
Not being super familiar with nix, does this mean that this flake will this only work on `x86_64-darwin` systems (Mac OS X on Intel chips)?  Ideally, `awssh` should remain platform-agnostic as much as possible.

Reply:
You are right. Fixed.

![image](https://github.com/cisagov/awssh/assets/6789916/b2fd3565-9398-45cb-93c7-ef30b15833bb)